### DISCUSSION TITLE
Option to swap Num layer

### DISCUSSION THREAD

**Discussion by Mithrandir2k18** [manna-harbour/miryoku#288] [Q&A] [answered]
An option to swap [the num layer](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#num) left and right hand side would be great I think. At least as a former 100% user who enjoyed using the num-pad from time to time with the right hand it seems kind of arbitrary to have to relearn this with the left hand.

**Comment by manna-harbour** [comment, marked as answer]
Learning a left hand numpad arrangement is easier than you might expect.

It's on the left so that Nav can be on the right for maximum dexterity, and both Num and Nav can be activated by primary (home) thumb keys. Also, all same-hand layers are based on the same arrangement.

There is a [Flip Layers option](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#flip) included that swaps all layers as well as Base layer thumbs between hands.

If you have muscle memory for right hand numpad and left hand ESDF nav then [Flip Layers with inverted-T Nav](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#flip-layers-inverted-t-nav) might be a good option.

You can also [add, remove, and substitute custom layers](https://github.com/manna-harbour/miryoku/discussions/85).
